packages/gatsby-image/src/index.js

```diff
@@ -165,7 +165,7 @@ function getIO() {
   return io
 }
 
-function generateImageSources(imageVariants) {
+function generateImageSources(imageVariants, imageMediaType) {
```

Comment:
Is there any benefit by using a parameter here with the logic extracted into fixed/fluid render branches?

This attribute assumes all images that the `<source>` provides are of the same image type, so why not just use the `src` here from each `imageVariant`?